doc/nrf/ug_matter.rst

```diff
@@ -50,32 +50,122 @@ The |NCS|'s Multiprotocol Service Layer (MPSL) driver allows running Bluetooth L
    nRF Connect platform in Matter
 
 For detailed description, see the `nRF Connect platform overview`_ Matter documentation page.
+If you are new to Matter, check also the tutorials on `DevZone`_.
+
+Matter development environment
+******************************
+
+When you program any of the available Matter samples to the supported development kits, you automatically build Matter's nRF Connect platform.
+The development kit and Matter's nRF Connect platform that is programmed on the development kit form the Matter accessory device.
+
+The default use case of a Matter accessory device is within a Matter network that is built on top of a low-power, 802.15.4-compatible Thread network.
+To pair and control the Matter accessory device remotely over such a network, you need to set up a Matter controller on a mobile or a PC.
+A Matter controller is a role within the Matter environment.
+This controller interacts with the accessory devices using the following protocols:
+
+* Bluetooth LE during the commissioning process - to securely pass the network credentials and provision the accessory device into the Thread network.
+* Regular IPv6 communication after the accessory device joins the Thread network - to interact with each other by exchanging application messages.
+  For example, to report temperature measurements of a sensor.
+
+The following figure shows the protocol types and the available Matter controllers.
+
+.. figure:: images/matter_protocols_controllers.svg
+   :scale: 50%
+   :alt: Protocol types and controllers used by Matter
+
+   Protocol types and controllers used by Matter
+
+To enable the IPv6 communication with the Matter accessory device over the Thread network, the Matter controller requires the Thread Border Router.
+This is because the Matter controller does not have an 802.15.4 Thread interface.
```

Comment:
pendantic: In theory the Matter controller can be a Thread device as well, so a more accurate statement would be that the Matter controller **typically** doesn't have an 802.15.4 Thread interface.

Reply:
I agree, in original docs, there was a word "usually". Let's be future-proof then ;)

Reply:
Can we be more specific what is hiding behind "typically"/"usually"?